include/robot_localization/navsat_transform.h

```diff
@@ -165,6 +165,11 @@ class NavSatTransform
     //!
     bool broadcast_cartesian_transform_as_parent_frame_;
 
+    //! @brief The cartesian frame ID, default as 'local_enu' if using Local Cartesian or 'utm' if using the UTM
+    //! coordinates as our cartesian.
+    //!
+    std::string cartesian_frame_id_;
```

Comment:
Would you mind putting this down with the other strings? I was aiming for grouping by datatype and _then_ alphabetically, though it appears there are some anomalies here.

Reply:
done!